staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -380,7 +384,14 @@ func (t *tracker) add(gvr schema.GroupVersionResource, obj runtime.Object, ns st
 			return err
 		}
 		if oldMeta.GetNamespace() == newMeta.GetNamespace() && oldMeta.GetName() == newMeta.GetName() {
-			if replaceExisting {
+			if replaceExisting && !reflect.DeepEqual(obj, existingObj) {
+				if t.versioner.CompareResourceVersion(obj, existingObj) < 0 {
```

Comment:
higher rv also results conflict

Reply:
Thanks! Fixed in https://github.com/kubernetes/kubernetes/pull/72353/commits/e6e477c505f51e4f1a4c7e896b1485d9948d931d.